adr/0017-hardware-screening-os.md

```diff
@@ -0,0 +1,34 @@
+# 0017. Hardware screening for OS
+
+Date: 2020-10-23
+
+## Status
+
+Draft
+
+## Context
+
+We need take a decision for the minimal requirements of a IoT-Board to become a Home Assistant OS image.
```

Comment:
Since every extra IoT board adds extra ongoing maintenance burden, is this a list of requirements that hardware needs to meet before it can be suggested? Or would any PR that adds support for hardware that meets these requirements be accepted?

Reply:
That is an interesting aspect. Just the machine build would be difficult at this point. They can run only generic qemu builds

Reply:
However, I will avoid the problem that we had with OrangePi. At least the required points need to exist that there is no maintenance burden for us.